assets/sass/components/user-input/googlesitekit-user-input-questions.scss

```diff
@@ -49,15 +49,17 @@
 		position: relative;
 	}
 
-	.googlesitekit-user-input__question-number {
+	p.googlesitekit-user-input__question-number {
 		color: $c-interactive-on-disable-container;
 		font-family: $f-secondary;
```

Comment:
@techanvil Looking at Darren's comment in QA, should this be `$f-primary`? I just checked the Figma mock myself and it is Google Sans Text (instead of Google Sans Display).

Reply:
Thanks for spotting @jimmymadon! I had completely missed that detail. Have updated :+1: